lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -154,6 +154,8 @@ def call(params)
                 filters << row.to_hash
               end
             end
+            fail 'The filter set can not be empty when using sync_multiple_projects_* mode as the filters contain the project ids \
```

Comment:
Style/MultilineIfModifier: Favor a normal if-statement over a modifier clause in a multiline statement.